src/Render/RenderOpenApi.php

```diff
@@ -43,12 +45,18 @@ public function __construct(ContainerInterface $generatorLocator, ?OpenApiRender
         }
     }
 
+    /**
+     * @return string[]
+     */
     public function getAvailableFormats(): array
     {
         return array_keys($this->openApiRenderers);
     }
 
-    public function renderFromRequest(Request $request, string $format, $area, array $extraOptions = [])
+    /**
+     * @param array<string, mixed> $extraOptions
+     */
+    public function renderFromRequest(Request $request, string $format, string $area, array $extraOptions = []): string
```

Comment:
class is not final or internal, this would be a breaking change when extending & overwriting the method. https://3v4l.org/oukC0

Parameter type can be added without breaking. https://3v4l.org/o8nDR

```suggestion
    /**
     * @param array<string, mixed> $extraOptions
     *
     * @return string
     */
    public function renderFromRequest(Request $request, string $format, string $area, array $extraOptions = [])
```